includes/functions/featured.php

```diff
@@ -9,81 +10,92 @@
  * @version $Id: featured.php 18695 2011-05-04 05:24:19Z drbyte $
  */
 
-////
-// Set the status of a featured product
-  function zen_set_featured_status($featured_id, $status) {
-    global $db;
-    $sql = "update " . TABLE_FEATURED . "
-            set status = '" . (int)$status . "', date_status_change = now()
-            where featured_id = '" . (int)$featured_id . "'";
-
-    return $db->Execute($sql);
-   }
-
-////
-// Auto expire products on featured
-  function zen_expire_featured() {
-    global $db;
-
-    $date_range = time();
-    $zc_featured_date = date('Ymd', $date_range);
-
-    $featured_query = "select featured_id
-                       from " . TABLE_FEATURED . "
-                       where status = '1'
-                       and ((" . $zc_featured_date . " >= expires_date and expires_date != '0001-01-01')
-                       or (" . $zc_featured_date . " < featured_date_available and featured_date_available != '0001-01-01'))";
-
-    $featured = $db->Execute($featured_query);
-
-    if ($featured->RecordCount() > 0) {
-      while (!$featured->EOF) {
-        zen_set_featured_status($featured->fields['featured_id'], '0');
-        $featured->MoveNext();
-      }
+/**
+ * Set the status of a featured product
+ * 
+ * @global object $db
+ * @param int $featured_id
+ * @param int $status
+ */
+function zen_set_featured_status(int $featured_id, int $status)
+{
+  global $db;
+  $sql = "UPDATE " . TABLE_FEATURED . "
+          SET status = " . (int)$status . ",
+              date_status_change = now()
+          WHERE featured_id = " . (int)$featured_id;
```

Comment:
Should this update include a 'LIMIT 1' for performance?

Reply:
Probably, yes.  And probably similar for all such pages.
But this PR is primarily about design changes, particularly making it consistent with design on other admin pages.

So, a separate PR for optimizing those update queries would be appropriate.

Reply:
OK.  When I've got time, I'll start that/those reviews.

Reply:
No rush. Probably simpler after these PRs are merged anyway, to avoid merge conflicts.